pyproject.toml

```diff
@@ -53,11 +54,11 @@ dependencies = [
 ]
 
 [tool.hatch.envs.default.scripts]
-test = "pytest -n auto --ignore=demo --dist=loadgroup {args:tests}"
+test = "pytest --ignore=demo --dist=loadgroup {args:tests}"
```

Comment:
This was done to workaround https://github.com/wheel-next/wheel-stub/issues/9 as well.